src/frontend/src/pages/PIADetailPage/index.tsx

```diff
@@ -0,0 +1,187 @@
+  }, [id, navigate, win]);
+
+  const handleAlertClose = () => {
+    setFetchPiaError('');
+  };
+  return (
+    <div className="bcgovPageContainer background ">
+      <div className="container__padding-inline ppq-form-section form__container row">
+        {fetchPiaError && (
+          <Alert
+            type="danger"
+            message="Something went wrong. Please try again."
+            onClose={handleAlertClose}
+            className="mt-2"
+          />
+        )}
+        <div className="form__title">
+          <h1>{pia.title}</h1>
+          <a href="/pia-edit" className="bcgovbtn bcgovbtn__primary">
+            <FontAwesomeIcon icon={faFileArrowDown} />
+          </a>
+        </div>
+        <div>
+          <div className="row">
+            <div className="col col-md-4">Status</div>
+            <div className="col col-md-4">Submitted on</div>
+            <div className="col col-md-4">Last modified</div>
+          </div>
+          <div className="row">
+            <div className="col col-md-4">
+              {pia.status ? pia.status : 'Submitted'}
+            </div>
+            <div className="col col-md-4">{dateToString(pia.createdAt)}</div>
+            <div className="col col-md-4">{dateToString(pia.updatedAt)}</div>
+          </div>
+        </div>
+        <div className="horizontal-divider"></div>
+        <div className="container d-grid gap-3">
+          <h2>
+            <b>{messages.GeneralInfoSection.H2Text.en}</b>
+          </h2>
+          <div>
+            <div className="row ">
+              <div className="col col-md-4">
+                <b>Drafter</b>
+              </div>
+              <div className="col col-md-4">
+                <b>Initiative Lead</b>
+              </div>
+              <div className="col col-md-4">
+                <b>Ministry Privacy Officer </b>
+              </div>
+            </div>
+            <div className="row">
+              <div className="col col-md-4">{pia.drafterName}</div>
+              <div className="col col-md-4">{pia.leadName}</div>
+              <div className="col col-md-4">{pia.mpoName}</div>
+            </div>
+            <div className="row">
+              <div className="col col-md-4">{pia.drafterTitle}</div>
+              <div className="col col-md-4">{pia.leadTitle}</div>
+              <div className="col col-md-4">{pia.mpoEmail}</div>
+            </div>
+            <div className="row">
+              <div className="col col-md-4">{pia.drafterEmail}</div>
+              <div className="col col-md-4">{pia.leadEmail}</div>
+            </div>
+          </div>
+          <div>
+            <div className="row">
+              <div className="col col-md-4">
+                <b>Ministry</b>
+              </div>
+              <div className="col col-md-4">
+                <b>Branch</b>
+              </div>
+            </div>
+            <div className="row">
+              <div className="col col-md-4">{pia.ministry}</div>
```

Comment:
this should be the name of the ministry, and not the enum returned value from the backend

Reply:
good catch, i will change to full name of ministry